MIGRATION_GUIDE.md

```diff
@@ -0,0 +1,60 @@
+# v0.x.x &rightarrow; v1.0.0-beta.0
+
+Full documentation for v1 endpoints is available here: https://paritytech.github.io/substrate-api-sidecar/dist/
+
+For most users, the major bump will require transitioning all the endpoint paths
+they use to an updated version. The v1.0.0.beta.0 release contains both the old
+and new endpoints to help facilitate the transition.
+
+**`substrate-api-sidecar` releases after Friday, October 2nd, 2020, will not
+make any gurantees to include deprecated endpoints.** It is important that users
+finishing transitioning to the new endpoints by this date so they are ready for
+any subsequent emergency updates.
+
+Below is a table that is intended to give an overview of what the mapping of
+deprecated endpoints to there v1 equivalents. Many of the new
+endpoints have unlisted options for query params.
```

Comment:
Add a note that some of the response objects are different too. Or maybe add a symbol on the v1 side of the table, like `*` to indicate that the response has a breaking change.

Edit: Oh, you cover it after

Reply:
"unlisted"? Can you elaborate on what that means?

Reply:
I removed "unlisted" and rephrased it to basically say "go look at the docs for more info on query params".